anyway/widgets/suburban_widgets/motorcycle_accidents_vs_all_accidents_widget.py

```diff
@@ -163,5 +166,12 @@ def localize_items(request_params: RequestParams, items: Dict) -> Dict:
         }
         return items
 
+    # noinspection PyUnboundLocalVariable
+    def is_included(self) -> bool:
+        motor_accidents_in_road = self.items[0]["series"][0]["value"]
+        motor_accidents_in_all_roads = self.items[1]["series"][0]["value"]

```

Comment:
I think this might not be best practice.
Perhaps it's better to save additional data while generating the items that will be used in the in_included function (instead of using `self.items[0]["series"][0]["value"]` or `self.items[1]["series"][0]["value"]`